package.json

```diff
@@ -255,9 +255,10 @@
   },
   "devDependencies": {
     "@types/node": "^14.14.41",
+    "@types/semver": "^7.7.0",
```

Comment:
I added `semver` to make parsing the rescript version easier.

If adding this dependency is fine, then I can also refactor [`incrementalCompilation.figureOutBscArgs`](https://github.com/rescript-lang/rescript-vscode/blob/f4ba82955050e66bd678139f0a6e7888251219e1/server/src/incrementalCompilation.ts#L576) and [`utils.runAnalysisAfterSanityCheck`](https://github.com/rescript-lang/rescript-vscode/blob/f4ba82955050e66bd678139f0a6e7888251219e1/server/src/utils.ts#L211-L220) to use it.

https://github.com/rescript-lang/rescript-vscode/blob/f4ba82955050e66bd678139f0a6e7888251219e1/server/src/incrementalCompilation.ts#L576

https://github.com/rescript-lang/rescript-vscode/blob/f4ba82955050e66bd678139f0a6e7888251219e1/server/src/utils.ts#L211-L220

Reply:
I've done the proposed refactoring in this commit: https://github.com/rescript-lang/rescript-vscode/pull/1093/commits/d4cc19db8509a9f0273600fc955a839cfa7b0712

Reply:
Great! 👍